tests/store/handlers/test_readhandler.py

```diff
@@ -135,6 +135,42 @@ def test_get_non_archived_files(
     assert non_archived_file in archived_files
 
 
+def test_get_bundle_name_from_file_path(
+    populated_store: Store, sample_id: str, spring_file_1: Path
+):
+    """Test that the bundle name is fetched correctly from a file path."""
+    # GIVEN a store containing a spring file related to sample ACC123456A1
+
+    # WHEN getting the bundle name for the file
+    bundle_name: str = populated_store.get_bundle_name_from_file_path(spring_file_1.as_posix())
+
+    # THEN the bundle name should be the sample name
+    assert bundle_name == sample_id
+
+
+def test_get_all_non_archived_spring_files(populated_store: Store):
+    """Test that getting all non-archived spring files from the store
+    returns all files fulfilling said condition."""
+    # GIVEN a populated store containing four files
+
+    # WHEN retrieving all non archived spring files and compare to all files
+    non_archived_spring_files: List[File] = populated_store.get_all_non_archived_spring_files()
+    all_files: List[File] = populated_store.get_files().all()
+
+    # THEN both queries should contain entries
+    assert non_archived_spring_files
+    assert all_files
```

Comment:
Rewrote for adjusting better to the given, when, then structure
```suggestion
    # GIVEN a populated store containing four files
    all_files: List[File] = populated_store.get_files().all()
    assert len(all_files) == 4

    # WHEN retrieving all non-archived spring files and compare to all files
    non_archived_spring_files: List[File] = populated_store.get_all_non_archived_spring_files()
    
    # THEN the retrieved query is not empty
    assert non_archived_spring_files
    
```

Reply:
I generally don't like test that depend on the size of the underlying store fixture. I changed the doc string and restructured a bit